src/moscot/problems/_utils.py

```diff
@@ -52,30 +53,38 @@ def handle_cost(
     x: Optional[Mapping[str, Any]] = None,
     y: Optional[Mapping[str, Any]] = None,
     cost: Optional[Union[CostFn_t, Mapping[str, CostFn_t]]] = None,
+    cost_kwargs: Union[Mapping[str, Any], Mapping[str, Mapping[str, Any]]] = types.MappingProxyType({}),
     **_: Any,
-) -> Tuple[Optional[Mapping[str, Any]], Optional[Mapping[str, Any]], Optional[Mapping[str, Any]]]:
+) -> Tuple[Dict[str, Any], Dict[str, Any], Dict[str, Any]]:
+    x = {} if x is None else dict(x)
```

Comment:
For consistency, would it be possible to remove the `Optional` for `xy/x/y` here as well?